README.md

```diff
@@ -27,8 +27,35 @@ docker run --name meross-ingestor \
 - ⚠️MEROSS_EMAIL
 - ⚠️MEROSS_PASSWORD
 
-Variables marked with `⚠️` are required
+Variables marked with ⚠️ are required
```

Comment:
Removed the `` so that that the explanation icon matched that used in the main body.